src/gui/folder.cpp

```diff
@@ -217,9 +231,24 @@ bool Folder::checkLocalPath()
 
     QString error;
     if (fi.isDir() && fi.isReadable() && fi.isWritable()) {
-        qCDebug(lcFolder) << "Checked local path ok";
-        if (!_journal.open()) {
-            error = tr("%1 failed to open the database.").arg(_definition.localPath());
+        if (Utility::isWindows()) {
+#ifndef MAX_PATH
+#define MAX_PATH 260 // This is only here to make the code compile on non-windows machines. This whole if-statement will get removed in release builds on those platforms.
```

Comment:
Is that really much better than just an ifdef WIN

Reply:
I now put it in an `#ifdef`